src/_tutorials/server/get-started.md

```diff
@@ -1,263 +1,237 @@
-   using files from the template named
-   **console-full**, which has the description 
-   **A command-line application sample.**
+## 5. Run the app
 
-1. If the IDE doesn't automatically download the packages
-   that the app depends on, make the IDE run `pub get`.
+First we need to get the dependencies of the app using the
+[`pub`](/tools/pub/cmd) command:
 
-If you're using the command line, follow these instructions:
+```terminal
+> pub get
+```
 
-1. Get [Stagehand.]({{site.pub}}/packages/stagehand)
+To run the app from the command line, use the Dart VM by running the
+[`dart`][/tools/dart-vm] command:
 
-1. Use Stagehand to get the files, and then
-   use the pub tool to download dependent packages.
-   The commands you use should be something like this:
-   
-   ```terminal
-   mkdir hello_world
-   cd hello_world
-   stagehand console-full
-   pub get
-   ```
+```terminal
+> dart bin/main.dart
+Hello world: 42!
+```
 
-Some of the files and directories in the hello_world app
-include the following:
+## 6. Add custom code to the app
 
-`analysis_options.yaml`
-: Defines lint rules enforced for this project.
-  By default, this file specifies the set of rules used for Google projects,
-  but you can [customize these options](/guides/language/analysis-options).
+Let's customize the app you just created:
 
-`bin`
-: Contains the source files for the app.
-  The main Dart file for this example is `bin/main.dart`.
+ 1. Edit `lib/cli.dart` to return a different result:
 
-`lib`
-: Contains library code.
-  In this directory, `hello_world.dart` defines
-  a simple `calculate()` method.
+    ```dart
+    int calculate() {
+      return -1;
+    }
+    ```
+ 1. Save your changes.
 
-`pubspec.lock`
-: A generated file that specifies the version numbers
-  of the packages on which the app depends.
+ 1. Re-run the main entry-point of your app:
 
-`pubspec.yaml`
-: Declares which packages your app needs.
+    ```terminal
+    > dart bin/main.dart
+    Hello world: -1
+    ```
 
-`test`
-: Contains test code for the hello_world app.
+More information:
+[Write Command-Line Apps](/tutorials/server/cmdline)
 
-`.dart_tool`
-: Support files used by `pub` and other Dart tools. You can ignore this.
+## 7. Use DevTools to debug the app
 
-`.packages`
-: Tells the Dart tools where to get the packages that your
-  app uses. This file is created by the `pub get` command.
-  You can ignore this.
+If you wish to debug the app, you can use [Dart DevTools](/tools/dart-devtools).
 
+## 8. Compile for production
 
-## Run a command-line app {#run-cmd-line}
+The steps above used `dart`, the Dart VM to run the app. The Dart VM is
+optimized for fast, incremental compilation to provide instant feedback
+duringdevelopment. Now that our small app is done, it's time to AOT compile our
+Dart code to optimized native machine code.
 
-Run the app, using either your IDE or the command line:
+Use the `dart2aot` tool to AOT-compile the program to machine code:
```

Comment:
We aren't even putting `dart2aot` in the changelog. (See https://github.com/dart-lang/sdk/issues/36758.) It feels very wrong to put it into this tutorial, unless this page isn't linked to by anything.

Reply:
I added a page for `dart2aot` here: https://github.com/dart-lang/site-www/pull/1500/files#diff-a05d7dd79fd05505679e3fa9f09c618e

Reply:
> We aren't even putting dart2aot in the changelog

Send you a CL.